src/widgets/PopoverDropDown.js

```diff
@@ -65,7 +67,7 @@ const DropDown = ({ anchorRef, isVisible, options, onSelection, onClose, title }
       onRequestClose={onClose}
       arrowStyle={arrowStyle}
       backgroundStyle={popoverBackgroundStyle}
-      placement="bottom"
+      placement="auto"
```

Comment:
perhaps the component should change its name from `PopoverDropDown`?? 😆

Reply:
🤣  hmm.. `PopoverDropAuto` just doesn't seem to have the same ring to it